README.md

```
@@ -20,18 +20,472 @@ If you're using WebSocket, you don't need to worry about heartbeat request/respo
+
+**Attribute Management**
+- **`GetAttribute(key any)`**: Get session attribute
+- **`SetAttribute(key any, value any)`**: Set session attribute
+- **`RemoveAttribute(key any)`**: Remove session attribute
+
+**Statistics**
+- **`Stat()`**: Get session statistics (connection status, read/write bytes, packet count, etc.)
+
+#### Active Time Update Mechanism
+
+**Automatic Active Time Updates**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)